authors/apps/articles/views.py

```diff
@@ -50,3 +50,40 @@ def destroy(self, *args, **kwargs):
         return Response(
             {"message": error_messages['delete_msg'].format('Article')},
             status=status.HTTP_200_OK)
+
+
+class ArticleLikesView(generics.CreateAPIView):
+    permission_classes = (IsAuthenticated,)
+
+    def put(self, request, slug):
```

Comment:
@calvinpete this function has a cognitive complexity of 6 exceeding the allowed 5. Please refactor

Reply:
@ja-odur  please share a  resource  on cognitive complexity

Reply:
@ja-odur I have resolved it